### PR TITLE
BZ1888035: Replace MCG module with link to OCS doc

### PR DESCRIPTION
**Pull request by apinnick**
https://bugzilla.redhat.com/show_bug.cgi?id=1888035

Update MCG module with links to OCS docs.

4.6+

Preview:

https://deploy-preview-35109--osdocs.netlify.app/openshift-enterprise/latest/migration_toolkit_for_containers/installing-mtc.html#migration-configuring-mcg_installing-mtc

QE approved. Needs peer review.

### CODE REVIEW

**Comment by XinRedhat** [comment]
@apinnick LGTM.

File: migrating_from_ocp_3_to_4/installing-3-4.adoc [hunk 1]
```diff
@@ -33,15 +33,15 @@ If you use a proxy server with an internally hosted replication repository, you
...
 The following storage providers are supported:
 
-* Multi-Cloud Object Gateway (MCG)
-* Amazon Web Services (AWS) S3
-* Google Cloud Platform (GCP)
-* Microsoft Azure Blob
+* xref:../migrating_from_ocp_3_to_4/installing-3-4.adoc#migration-configuring-mcg_installing-3-4[Multicloud Object Gateway]
+* xref:../migrating_from_ocp_3_to_4/installing-3-4.adoc#migration-configuring-aws-s3_installing-3-4[Amazon Web Services S3]
+* xref:../migrating_from_ocp_3_to_4/installing-3-4.adoc#migration-configuring-gcp_installing-3-4[Google Cloud Platform]
+* xref:../migrating_from_ocp_3_to_4/installing-3-4.adoc#migration-configuring-azure_installing-3-4[]Microsoft Azure Blob
```
Comment (bergerhoffer) [inline migrating_from_ocp_3_to_4/installing-3-4.adoc:39]: The ending backtick should be after the text (xref isn't rendering properly)

File: migrating_from_ocp_3_to_4/installing-restricted-3-4.adoc [hunk 2]
```diff
@@ -35,12 +35,12 @@ If you use a proxy server with an internally hosted replication repository, you
 
 [id="{context}_configuring-replication-repository-additional-resources"]
 [discrete]
-=== Additional resources
+=== Additional resources for configuring a replication repository
 
+* link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.7/html-single/planning_your_deployment/index#disconnected-environment_rhocs[Disconnected environment] in _Red Hat OpenShift Container Storage: Planning your deployment_.
```
Comment (bergerhoffer) [inline migrating_from_ocp_3_to_4/installing-restricted-3-4.adoc:40]: Not critical, but the way that this is worded makes me think that the book title is "Red Hat OpenShift Container Storage: Planning your deployment".

I'd consider separating the product from the book title. Could do something like:

```
in the _Planning your deployment_ guide for Red Hat OpenShift Container Storage.
```
or could simply omit the book title: 
```
in the Red Hat OpenShift Container Storage documentation.
```

File: migration_toolkit_for_containers/installing-mtc-restricted.adoc [hunk 3]
```diff
@@ -24,13 +24,22 @@ include::modules/migration-installing-mtc-on-ocp-4.adoc[leveloffset=+1]
...
 
 All clusters must have uninterrupted network access to the replication repository.
 
 If you use a proxy server with an internally hosted replication repository, you must ensure that the proxy allows access to the replication repository.
 
+[id="{context}_configuring-replication-repository-additional-resources"]
+[discrete]
+=== Additional resources for configuring a replication repository
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.7/html-single/planning_your_deployment/index#disconnected-environment_rhocs[Disconnected environment] in _Red Hat OpenShift Container Storage: Planning your deployment_.
```
Comment (bergerhoffer) [inline migration_toolkit_for_containers/installing-mtc-restricted.adoc:39]: Same comment about the book title

File: migrating_from_ocp_3_to_4/installing-3-4.adoc (hunk 1 above)
Comment (apinnick) [reply migrating_from_ocp_3_to_4/installing-3-4.adoc:39]: Oops-- sorry, missed that!

File: migrating_from_ocp_3_to_4/installing-restricted-3-4.adoc (hunk 2 above)
Comment (apinnick) [reply migrating_from_ocp_3_to_4/installing-restricted-3-4.adoc:40]: Great. I was not sure about conventions for this kind of reference.

File: migration_toolkit_for_containers/installing-mtc-restricted.adoc (hunk 3 above)
Comment (apinnick) [reply migration_toolkit_for_containers/installing-mtc-restricted.adoc:39]: Done. Also updating version to 4.8 now that OCS has released that version.

**Pull request merged**